src/app/technical-record/vehicle-summary/vehicle-summary-psv/vehicle-summary-psv.component.html

```diff
@@ -1,4 +1,64 @@
 <dl class="govuk-summary-list">
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Speed limiter exempt
+    </dt>
+    <dd id="test-speedLimiterMrk" class="govuk-summary-list__value">
+      {{ activeRecord.speedLimiterMrk | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Tacho exempt
+    </dt>
+    <dd id="test-tachoExemptMrk" class="govuk-summary-list__value">
+      {{ activeRecord.tachoExemptMrk | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Euro standard
+    </dt>
+    <dd id="test-euroStandard" class="govuk-summary-list__value">
+      {{ activeRecord.euroStandard | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Fuel/propulsion system
+    </dt>
+    <dd id="test-fuelpropulsionsystem" class="govuk-summary-list__value">
+      {{ activeRecord.fuelPropulsionSystem | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Number of wheels driven
+    </dt>
+    <dd id="test-numberOfWheelsDriven" class="govuk-summary-list__value">
+      {{ activeRecord.numberOfWheelsDriven | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
+  <div class="govuk-summary-list__row">
+    <dt class="govuk-summary-list__key">
+      Emission limit
+    </dt>
+    <dd id="test-emissionsLimit" class="govuk-summary-list__value">
+      {{ activeRecord.emissionsLimit | DefaultNullOrEmpty }}
+    </dd>
+    <dd class="govuk-summary-list__value"></dd>
+  </div>
+
```

Comment:
the spreadsheet combining the fields per vehicle types is showing fields only shown for psv as such:
emission limit, seats lower deck, seats upper deck, standing capacity, vehicle size, number of seat belts, seatbelt installation